spec/checker_spec.lua

```diff
@@ -531,5 +531,256 @@ describe("Titan type checker", function()
         local ok, err = run_checker(code)
         assert.truthy(ok)
     end)
+
+    for _, op in ipairs({"==", "~="}) do
+        it("can compare arrays of same type using " .. op, function()
+            local code = [[
+                function fn(a1: {integer}, a2: {integer}): boolean
+                    return a1 ]] .. op .. [[ a2
+                end
+            ]]
+            local ast, err = parser.parse(code)
+            local ok, err = checker.check(ast, code, "test.titan")
```

Comment:
Please use the `run_checker` function instead of calling `parser.parse` and `checker.check` — in PR #44 there is a change to `checker.check`, so using the helper function will abstract away that change and avoid conflicts

Reply:
Sorry, I just copied and pasted from an old file I had being working previously.